pkg/cmd/namespace/namespace.go

```diff
@@ -181,25 +191,105 @@ func (o *Options) lazyCreateNamespaceResource(ns string) error {
 }
 
 // UpdateNamespaceInYamlFiles updates the namespace in yaml files
-func UpdateNamespaceInYamlFiles(dir string, ns string, filter kyamls.Filter) error { //nolint:gocritic
-	modifyFn := func(node *yaml.RNode, path string) (bool, error) {
+func UpdateNamespaceInYamlFiles(rootDir string, dir string, ns string, filter kyamls.Filter, shouldMoveFiles bool) ([]string, error) { //nolint:gocritic
```

Comment:
`rootDir, dir, ns string` should fix the gocritic linter warning i think 🤔

Reply:
Yeah, I guess my IDE appended that nolint thing :thinking: